views/voicemails-rows.html

```diff
@@ -24,16 +24,16 @@
 			<div class="bottom-line">{{toFriendlyDate timestamp 'time'}}</div>
 		</td>
 		{{#if formatted.showCallerIDName}}
-			<td class="select-line no-padding" data-filter-value="{{from}} {{formatted.from}} {{ formatted.callerIDName }}">
+			<td class="select-line no-padding" data-filter-value="{{from}} {{formatted.from}} {{ formatted.callerIDName }}" data-sort-value="{{from}} {{formatted.from}} {{ formatted.callerIDName }}" >
 				<div class="top-line">{{#monsterNumberWrapper formatted.callerIDName}}{{/monsterNumberWrapper}}</div>
 				<div class="bottom-line">{{#monsterNumberWrapper formatted.from}}{{/monsterNumberWrapper}}</div>
 			</td>
 		{{else}}
-			<td class="select-line" data-filter-value="{{from}} {{formatted.from}} {{ formatted.callerIDName }}">
+			<td class="select-line" data-filter-value="{{from}} {{formatted.from}} {{ formatted.callerIDName }}" data-sort-value="{{from}} {{formatted.from}} {{ formatted.callerIDName }}">
 				{{#monsterNumberWrapper formatted.from}}{{/monsterNumberWrapper}}
 			</td>
 		{{/if}}
-		<td class="select-line" data-filter-value="{{to}} {{formatted.to}}">{{#monsterNumberWrapper formatted.to}}{{/monsterNumberWrapper}}</td>
+		<td class="select-line" data-filter-value="{{to}} {{formatted.to}}" data-data-value="{{to}} {{formatted.to}}">{{#monsterNumberWrapper formatted.to}}{{/monsterNumberWrapper}}</td>
```

Comment:
And here as well. In this case, you can pick between the two but it is usually better to pick the one that's displayed to the user.
```suggestion
		<td class="select-line" data-filter-value="{{to}} {{formatted.to}}" data-sort-value="{{formatted.to}}">{{#monsterNumberWrapper formatted.to}}{{/monsterNumberWrapper}}</td>
```
(also fixed a typo `data-data-value` -> `data-sort-value`)